modules/ivanchenko_a_metrics/src/metrics.cpp

```diff
@@ -0,0 +1,51 @@
+// Copyright 2024 Ivanchenko Aleksei
+
+#include "include/metrics.h"
+
+#include <cmath>
+
+float metrics::l1(std::vector<float> v1, std::vector<float> v2) {
```

Comment:
Лучше передавать векторы по константной ссылке (const reference), чтобы избежать копирования
```suggestion
float metrics::l1(const std::vector<float>& v1, const std::vector<float>& v2) {
```

Аналогично для остальных функций